src/main/java/edu/kit/compiler/transform/StandardLibraryEntities.java

```diff
@@ -0,0 +1,43 @@
+package edu.kit.compiler.transform;
+
+import firm.Entity;
+import firm.Ident;
+import firm.MethodType;
+import firm.Mode;
+import firm.PrimitiveType;
+import firm.Program;
+import firm.Type;
+import firm.bindings.binding_typerep.ir_visibility;
+import lombok.Getter;
+
+public enum StandardLibraryEntities {
```

Comment:
I'm not sure I understand why this is an enum. Do you intend this class to be used statically or do you intend the instance to be passed as an argument to the code generation functions?

Also you probably need to ensure that Firm is initialized before/when the constructor is called.

Reply:
This is a simple way of expressing a Singleton in Java. The constructor is not called until the `Instance` entry is first accessed.

Reply:
Ok, once #67 is merged I will add that.